docker-compose.reputation.yml

```diff
@@ -8,7 +8,7 @@ services:
   web:
     environment:
       - "REPUTATION_SERVER=http://reputation-web-grant:3334"
-      - "REPUTATION_TOKEN=foobarfoobar"
+      - "REPUTATION_TOKEN=636765a9-1f92-4b40-ab0b-85ebd1e2c23d"
```

Comment:
i would say yes, learning from using multiple tokens with multiple different servers, if all of the tokens are the same then it would be difficult to suss out why one is failing. basically things brings our tests closer to reality at near zero cost

Reply:
reverting.

Reply:
https://github.com/brave-intl/bat-go/pull/314/commits/ee91546b83628e93123027d39ade0ef864f1a4bb

Reply:
thanks, was just curious how the change came about